network-api/networkapi/wagtailpages/pagemodels/customblocks/link_block.py

```diff
@@ -34,27 +39,23 @@ class LinkBlock(BaseLinkBlock):
         required=False,
         label="Link to",
     )
+    anchor = blocks.CharBlock(
+        max_length=300,
+        required=False,
+        validators=[AnchorLinkValidator()],
+        label="#",
+        help_text='An id attribute of an element on the current page. For example, "#section-1"',
+    )
+    email = blocks.EmailBlock(required=False)
     file = DocumentChooserBlock(required=False, label="File")
     phone = blocks.CharBlock(max_length=30, required=False, label="Phone")
 
-    label = blocks.CharBlock()
-
     new_window = blocks.BooleanBlock(label="Open in new window", required=False)
 
-    def __init__(self, local_blocks=None, **kwargs):
-        super().__init__(local_blocks, **kwargs)
-        # Reoder child_blocks so that label is first
-        self.child_blocks = self.base_blocks.copy()
-        child_blocks = OrderedDict(
-            {
-                "label": self.child_blocks.pop("label"),
-            }
-        )
-        child_blocks.update({k: v for k, v in self.child_blocks.items()})
-        self.child_blocks = child_blocks
```

Comment:
Double-checking that this __init__ method isn't needed. I noticed it wasn't moved with the label to `BaseLinkBlock`.

Doesn't seem necessary with how that's been organized now, but asking for clarification because in retrospect maybe it wasn't needed in the first iteration?

Reply:
Yep, now that the label is in the right place we can remove this `__init__` method :+1:

Reply:
Awesome, thanks!